libs/backend/dto/src/dtos/map/map-review-suggestions.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { Gamemode, MapReviewSuggestions } from '@momentum/constants';
+import { IsNumber, IsString } from 'class-validator';
+
+export class GamemodeSuggestionsDto {
+  @IsNumber()
+  tier: number;
+
+  @IsString()
+  comment: string;
+
+  @IsNumber()
+  gameplayRating: number;
+}
+
+export class MapReviewSuggestionsDto implements MapReviewSuggestions {
+  [track: number]: { [x in Gamemode]: GamemodeSuggestionsDto };
+}
```

Comment:
We need to figure out a better way of doing this but I'm too tired at the moment, will think about it tomorrow.

Reply:
Okay, we can maybe transform this into a Map, which sounds silly but class-validation is constantly doing stupid transformations at runtime anyway. Let's leave this for now, I'll make a small issue to come back to this and work on it myself.